Samples/Xamarin/HelloWorld/HelloWorld.Ninject/HelloWorld.Ninject/Bootstrapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using HelloWorld.Views;
+using Prism.Ninject;
+using Xamarin.Forms;
+using Ninject;
+
+namespace HelloWorld
+{
+    public class Bootstrapper : NinjectBootstrapper
+    {
+        protected override Page CreateMainPage()
+        {
+            return Kernel.Get<MainPage>();
+        }
+
+        protected override void RegisterTypes()
+        {
+            Kernel.RegisterTypeForNavigation<ViewB>();
+            Kernel.RegisterTypeForNavigation<ViewA>();
```

Comment:
This bootstrapper is pretty much the same as the Unity version except it derives from NinjectBootstrapper and uses a Kernel in the Ninject fashion.